.github/workflows/crucible-ci.yaml

```diff
@@ -9,29 +9,83 @@ on:
   workflow_dispatch:
 
 jobs:
-  crucible-ci:
-    runs-on: [self-hosted, cpu-partitioning, remotehost]
+  generate-job-matrix-parameters:
+    runs-on: ubuntu-latest
+    outputs:
+      userenvs: ${{ steps.get-userenvs.outputs.userenvs }}
```

Comment:
steps.get-userenvs would be more intuitive... steps.get-userenvs.outputs.userenvs is weird

Reply:
I hear you, but this is the way it is.  Each `step` can have one or more outputs defined so the are "collected" under a "container"/"object" called "outputs" (my poor attempt at describing how it works).